distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -887,7 +896,7 @@ public void testCachedContentLengthHeader() throws IOException {
                 .headers();
 
         //Get again to hit the cache
-        get(url).then()
+        get(url + "&mode=cache").then()
```

Comment:
Shouldn't it automatically hit the cache the second time without the need for the `mode=cache` parameter?